test/e2e/multikueue/e2e_test.go

```diff
@@ -459,6 +460,88 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 				}, util.Timeout, util.Interval).Should(gomega.Succeed())
 			})
 		})
+
+		ginkgo.It("Should run a kubeflow PyTorchJob on worker if admitted", func() {
+			pyTorchJob := testingpytorchjob.MakePyTorchJob("pytorchjob1", managerNs.Name).
```

Comment:
Add a comment on the worker we expect o run it and why

 (nit: maybe we should make this one run in worker 2)

Reply:
make sense I will update in all of the jobs